src/components/map-container/map-container-component.vue

```diff
@@ -38,7 +39,7 @@ import 'mapbox-gl/dist/mapbox-gl.css';
  */
 
 // ⚠️PR REVIEWERS - Make sure this is commented out before merge!
-mapboxgl.accessToken = process.env.VUE_APP_MAPBOX_KEY as string;
+// mapboxgl.accessToken = process.env.VUE_APP_MAPBOX_KEY as string;
```

Comment:
Should this be commented out for the live site?

Reply:
Since we can deploy individual branches to heroku to test live functionality as needed, yeah we should still keep this commented out when merging to `develop`.